rpc/core/status.go

```diff
@@ -85,3 +91,15 @@ func validatorAtHeight(h int64) *types.Validator {
 	_, val := vals.GetByAddress(privValAddress)
 	return val
 }
+
+func GetNodeInfo(env *Environment, consensusParams cmtproto.ConsensusParams) p2p.DefaultNodeInfo {
```

Comment:
[suggestion] Would it make sense to move the retrieval of the consensus state inside this function to make it self-sufficient?
```go
func GetNodeInfo(env *Environment) (*p2p.DefaultNodeInfo, error) {
consensusParams, err := env.StateStore.LoadConsensusParams(latestHeight)
	if err != nil {
		return nil, err
	}
...
```

Reply:
Yea can do!

Reply:
Done!  f936054